release-notes/5.0/preview/api-diff/preview4/.Net/5.0-preview4_System.Runtime.Intrinsics.X86.md

```diff
@@ -0,0 +1,37 @@
+# System.Runtime.Intrinsics.X86
+
+``` diff
+ namespace System.Runtime.Intrinsics.X86 {
```

Comment:
System.Runtime.Intrinsics LGTM. CC. @CarolEidt @echesakovMSFT

Noting that the ARM intrinsics are part of System.Runtime.Intrinsics.Experimental still and aren't part of the public surface area.
We should hopefully wrap up the rest of the APIs in the next review and can then start moving them in box.